pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1234,7 +1240,7 @@ def _removeOldRootfs(self):
     def compile(self, **kwargs):
         # We could also just pass all values in KERNCONF to build all those kernels. However, if MFS_ROOT is set
         # that will apply to all those kernels and embed the rootfs even if not needed
-        super().compile(all_kernel_configs=self.kernelConfig, mfs_root_image=self.mfs_root_image, sysroot_only=self.sysroot_only, **kwargs)
```

Comment:
why did you make this change?

Reply:
Oops, I forgot to revert it from a previous set of changes that ended up not working.